src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -257,7 +257,11 @@ protected function getDateTimeWithOptionalFormat($format, $value)
     protected function getDateTime($value)
     {
         try {
-            return Date::parse($value);
```

Comment:
just replace this line by `return @Date::parse($value) ?: null;`. don't need the rest.

Reply:
Well, @GrahamCampbell you taught me something today. Thanks!